Sources/PackageManagerDocs/Documentation.docc/PackageSecurity.md

```diff
@@ -0,0 +1,50 @@
+# Package Security
+
+Learn about the security features that SwiftPM implements.
+
+## Trust on First Use
+
+SwiftPM records **fingerprints** of downloaded package versions so that
+it can perform [trust-on-first-use](https://en.wikipedia.org/wiki/Trust_on_first_use)
+(TOFU). 
+That is, when a package version is downloaded for the first time, SwiftPM trusts that 
+it has downloaded the correct contents and requires subsequent downloads of the same 
+package version to have the same fingerprint. 
+If the fingerprint changes, it might be an indicator that the package has been
+compromised and SwiftPM will either warn or return an error.
+
+Depending on where a package version is downloaded from, a different value is
+used as its fingerprint:
+                             
+| Package Version Origin | Fingerprint |
+| ---------------------- | ----------- |
+| Git repository         | Git hash of the revision |
+| Package registry       | Checksum of the source archive |
+
+All version fingerprints of a package are kept in a single file
+under the `~/.swiftpm/security/fingerprints` directory.
+  - For a Git repository package, the fingerprint filename takes the form of `{PACKAGE_NAME}-{REPOSITORY_URL_HASH}.json` (e.g., `LinkedList-5ddbcf15.json`).
+  - For a registry package, the fingerprint filename takes the form of `{PACKAGE_ID}.json` (e.g., `mona.LinkedList.json`).
+
+
+### Using fingerprints for TOFU
```

Comment:
Everything from line 30 (heading included) is replicated content from above - I made some suggestions to cherry pick a few details in the content above, and I think everything below can be deleted. Give it a read through and verify though.

Reply:
After re-reading, this makes sense - the only extra bit I've taken from `Using fingerprints for TOFU` is the part about converting the TOFU error to a warning using the `swift build` option `--resolver-fingerprint-checking`, wherein I link out to the `swift build` CLI article. Otherwise, all the content is essentially the same!